ProxyExtension/SplitTunnelDNSProxyProvider/SplitTunnelDNSProxyProvider.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import NetworkExtension
+
+final class SplitTunnelDNSProxyProvider : NEDNSProxyProvider {
+    override init() {
+        super.init()
+    }
+
+    override func startProxy(options:[String: Any]? = nil, completionHandler: @escaping (Error?) -> Void) {
+        completionHandler(nil)
+    }
+
+    override func stopProxy(with reason: NEProviderStopReason, completionHandler: @escaping () -> Void) {
+        completionHandler()
+    }
+
+    //  Be aware that by returning false in NEDNSProxyProvider handleNewFlow(),
+    //  the flow is discarded and the connection is closed
```

Comment:
but it's just closed on our side right? the network stack will then re-try the DNS request using the traditional means?

Reply:
I am testing this in my next ticket! 
Based on Apple docs it looks like the dns proxy works similarly to the "standard" app proxy, the base class for the transparent proxy we use for traffic split tunnel. If that is indeed the case, we will have to find a way to work around it.
Will keep you posted about what I find out